### ISSUE TITLE
Implementing cp in keras!

### ISSUE THREAD

**Issue by miladdona** [jacobgil/pytorch-tensor-decompositions#14]
I want to implement your code (CP) in keras. since you replace each convolution layer with four convolution layers in sequence, I am ok with the first and last layers, but I don't understand the two middle layers specially with the **groups** option of convolution layer. I have implemented it in keras using groups option but I get bad results in terms of accuracy.
Do you think the **groups** option of convolution layer in keras and pytorch is the same?
I have checked these two docs for keras and pytorch and it seems they are the same but I don't know why I cannot get a good accuracy!

https://github.com/christianversloot/machine-learning-articles/blob/main/grouped-convolutions-with-tensorflow-2-and-keras.md
https://pytorch.org/docs/stable/generated/torch.nn.Conv2d.html

thanks
